test/metabase/related_test.clj

```diff
@@ -130,6 +134,28 @@
        result-mask))
 
 
+;; We should ignore non-active entities
+
+(defn- exec! [& statements]
+  (doseq [statement statements]
+    (jdbc/execute! one-off-dbs/*conn* [statement])))
+
+(expect
+  [1 0]
+  (one-off-dbs/with-blank-db
+    (exec! "CREATE TABLE blueberries_consumed (num SMALLINT NOT NULL, weight FLOAT)")
+    (one-off-dbs/insert-rows-and-sync! (range 50))
+    (let [count-related-fields (fn []
+                                 (->> ((users/user->client :crowberto) :get 200
+                                       (format "field/%s/related" (data/id :blueberries_consumed :num)))
+                                      :fields
+                                      count))
+          before               (count-related-fields)]
+      (exec! "ALTER TABLE blueberries_consumed DROP COLUMN weight")
+      (sync/sync-database! (data/db))
+      [before (count-related-fields)])))
+
```

Comment:
Extra line

Reply:
The line is intentional to delineate blocks of functionality/related tests. Looking at other tests I was under the impression that this is the convention we're using. If not, no biggie, I'll change.